test/OpenTelemetry.Shims.OpenTracing.Tests/ListenAndSampleAllActivitySources.cs

```diff
@@ -6,6 +6,6 @@
 namespace OpenTelemetry.Shims.OpenTracing.Tests;
 
 [CollectionDefinition(nameof(ListenAndSampleAllActivitySources))]
-#pragma warning disable CA1515
+#pragma warning disable CA1515 // Consider making public types internal
 public sealed class ListenAndSampleAllActivitySources : ICollectionFixture<ListenAndSampleAllActivitySourcesFixture>;
-#pragma warning restore CA1515
+#pragma warning restore CA1515 // Consider making public types internal
```

Comment:
What's blocking us from making this `internal` ?

Reply:
xUnit1027 - Collection definition classes must be public. Add or change the visibility modifier of the collection definition class to public.